dandi/cli/cmd_download.py

```diff
@@ -18,6 +18,8 @@
     help=f"""\
 Download files or entire folders from DANDI.
 
+\b
+{_dandi_url_parser.resource_identifier_primer}
 \b
 {_dandi_url_parser.known_patterns}
```

Comment:
What about adding examples in how to get a folder or a file? Could be on 000027 released version